content/en/localstack/limitations.md

```diff
@@ -30,3 +30,40 @@ Limitations that may occur because of third party integrations behavior.
 By default, stacks with validated certificates may not be deployed using the `local` lambda executor.
 This originates from the way how CDK ensures the certificate is ready - it creates a single-file lambda function with a single dependency on `aws-sdk` which is usually preinstalled and available globally in lambda runtime.
 When this lambda is executed locally from the `/tmp` folder, the package can not be discovered by Node due to the way how Node package resolution works.
+
+### DNS Rebind Protection
+
+For certain LocalStack features it is necessary that the DNS resolves to the local network.
+For example, OpenSearch clusters which are created by LocalStack can be reached via `<domain-name>.<region>.opensearch.localhost.localstack.cloud`.
+This is handled correctly if you [configured your system's DNS for the transparent execution mode of LocalStack Pro]({{< ref "dns-server#system-dns-configuration" >}}).
+
+However, if you rely on your local network's DNS, your router / DNS server might block those requests due to the DNS Rebind Protection.
+This feature is enabled by default in OpenWRT, AVM FritzBox, and potentially also other devices.
```

Comment:
pfSense and OPNSense might be prominent examples as well, not sure if prominent enough to mention though, also many corporate firewalls. Its a default feature for unbound, and might be enabled by default (pfsense and opensense currently just set the private-addresses variable) there in the future as well (currently private-addresses is empty by default).

Reply:
Thanks! I added pfSense and OPNSense to the list of vendors. I won't go into detail on the differences though.